CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.7.0] - 2023-09-27
+### New
+- `PlatformClient.updateTermsOfServiceAccepted(bool)` to update the Terms of Service status (#91)
+- `Room.onConnectionFailed` to update the Room user on any SFU connection failure (#92)
+- `SFUConnection.isConnectionFailed` to be able to identify which SFUConnection has failed (#92)
```

Comment:
Did we add `SFUConnection.isConnectionFailed` or is this out of date?

Reply:
sorry! This was removed afterward and I forgot to remove it here.